dotcom-rendering/src/web/components/ContainerTitle.tsx

```diff
@@ -57,10 +51,6 @@ const bottomMargin = css`
 
 const marginStyles = css`
 	margin-left: 0;
```

Comment:
This brings the title in line with frontend: 

![Around the world](https://user-images.githubusercontent.com/102960844/218804089-15394c45-40e0-4fc5-87e0-f04c55c96291.png)

DCR before: 
![Around the world](https://user-images.githubusercontent.com/102960844/218804287-5880951c-761b-4504-81fd-816c8718cd2a.png)

DCR now: 
![image](https://user-images.githubusercontent.com/102960844/218804966-1050e380-95b5-4bba-bb63-417e3e9f4096.png)